Library/Homebrew/brew.sh

```diff
@@ -191,6 +191,16 @@ case "$HOMEBREW_COMMAND" in
   --config)    HOMEBREW_COMMAND="config" ;;
 esac
 
+if [[ -z "$HOMEBREW_DEVELOPER" ]]
+then
+  export HOMEBREW_GIT_CONFIG_FILE="$HOMEBREW_REPOSITORY/.git/config"
+  HOMEBREW_GIT_CONFIG_DEVELOPERMODE="$(git config --file="$HOMEBREW_GIT_CONFIG_FILE" --get homebrew.developermode)"
```

Comment:
Yeah, but that's compatible with not using the gitconfig trick.  I still don't see what the point of the gitconfig modifications are.  If we want certain cmds to always exhibit dev-modeish behavior, then those cmds should just stop checking for developerhood, rather than being tricked into thinking they're dealing with a developer.  The non-dev cmds can continue checking for developerhood.